### PR TITLE
Mobile App Development Section

### PR DESCRIPTION
**Pull request by akruj17**
Might be a little lengthy...

### CODE REVIEW

File: mobile-apps/README.md [hunk 1]
```diff
@@ -1,4 +1,191 @@
...
+https://reactnative.dev/docs/getting-started<br/>
+***Expo Documentation (Same as earlier)***<br/>
+https://docs.expo.io
+
+## Other Features
+The remainder of this document discusses some common features that get used in mobile apps, and we include tutorials on how to go about setting them up. All React Native resources should generally be assumed to be for the React Native CLI since the Expo documentation already does a great job with examples. 
+### Navigation
+One of the first things you will probably want to design is the flow of your app. This refers to how you can navigate to different screens, and what actions trigger transitions. All three forms of developments have components to simplify your navigation, such as tab bars, sliding drawers, navigation bars, and a variety of transitions effects. There is much to explore, but here are some tutorials to get started.
+***Creating Tab Bar Application in Native iOS***<br/>
+https://codewithchris.com/ios-tab-bar-app/
```
Comment (didizhou1) [inline mobile-apps/README.md:107]:
```suggestion
https://codewithchris.com/ios-tab-bar-app/<br/>
```

File: mobile-apps/README.md [hunk 2]
```diff
@@ -1,4 +1,191 @@
...
+***Create Navigation Drawer for Android App***<br/>
+https://code.tutsplus.com/tutorials/how-to-code-a-navigation-drawer-in-an-android-app--cms-30263
```
Comment (didizhou1) [inline mobile-apps/README.md:109]:
```suggestion
https://code.tutsplus.com/tutorials/how-to-code-a-navigation-drawer-in-an-android-app--cms-30263<br/>
```

File: mobile-apps/README.md [hunk 3]
```diff
@@ -1,4 +1,191 @@
...
+***React Native Official Documentation***<br/>
+https://reactnative.dev/docs/getting-started<br/>
+***Expo Documentation (Same as earlier)***<br/>
+https://docs.expo.io
+
+## Other Features
+The remainder of this document discusses some common features that get used in mobile apps, and we include tutorials on how to go about setting them up. All React Native resources should generally be assumed to be for the React Native CLI since the Expo documentation already does a great job with examples. 
+### Navigation
+One of the first things you will probably want to design is the flow of your app. This refers to how you can navigate to different screens, and what actions trigger transitions. All three forms of developments have components to simplify your navigation, such as tab bars, sliding drawers, navigation bars, and a variety of transitions effects. There is much to explore, but here are some tutorials to get started.
+***Creating Tab Bar Application in Native iOS***<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:106]:
```suggestion

***Creating Tab Bar Application in Native iOS***<br/>
```

File: mobile-apps/README.md [hunk 4]
```diff
@@ -1,4 +1,191 @@
...
+***Maps in React Native***<br/>
+https://www.freecodecamp.org/news/how-to-integrate-maps-in-react-native-using-react-native-maps-5745490fe055/<br/>
+
+
+### Accessing the Camera and Photos
+If uploading and downloading content to and from your app is important, then you might need to access a user’s camera and photo library. This is a feature that requires explicit permission from your users, so if you’re designing a fully-fledged app, make sure to have a backup route through your app in case permission isn’t granted. 
+
+***Native iOS Tutorial***<br/>
+https://www.zerotoappstore.com/how-to-access-camera-photo-library-in-swift.html<br/>
+***Make a Camera App in Swift (official Resource)***<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:147]:
```suggestion
***Make a Camera App in Swift (official resource)***<br/>
```

File: mobile-apps/README.md [hunk 5]
```diff
@@ -1,4 +1,191 @@
...
+***Native Android Tutorial***<br/>
+https://developer.android.com/training/volley<br/>
+***React Native Request Examples***<br/>
+https://reactnative.dev/docs/network<br/>
+
+### Storing Data Locally
+In some cases, your app might need to store local data that pertains only to the user of your app. You can use a local database for this, and there a variety of options listed below.  If you only need to store a few properties as key-value pairs, manipulating a list of user preferences is a much lighter option than creating a fully-fledged database.<br/>
+Keep in mind that if you want your user’s data to be easily accessible by other users or by other applications, using an online database is a much better solution. Check out our database section on options for getting this set up. 
+
+**Database Options**<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:168]:
```suggestion
#### Database Options 
```

File: mobile-apps/README.md [hunk 6]
```diff
@@ -1,4 +1,191 @@
...
+***Native iOS RealmSwift Tutorial***<br/>
+https://learnappmaking.com/realm-database-swift-getting-started/#getting-data-from-realm-database
+***Native Android Tutorial***<br/>
+https://developer.android.com/training/data-storage/room<br/>
+***React Native Tutorial Realm Documentation***<br/>
+https://realm.io/docs/javascript/latest/<br/>
+***React Native Realm Tutorial***<br/>
+https://aboutreact.com/example-of-realm-database-in-react-native/#Code-for-Realm-Database<br/>
+
+**User Preference Options**<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:185]:
```suggestion
#### User Preference Options
```

File: mobile-apps/README.md [hunk 7]
```diff
@@ -1,4 +1,191 @@
...
+
+**Database Options**<br/>
+These are just a few of the many database options available to use.<br/>
+***Native iOS Core Data Documentation: Core Data is Apple's official local database solution***<br/>
+https://developer.apple.com/documentation/coredata<br/>
+***Native iOS Core Data Tutorial***<br/>
+http://www.brianadvent.com/build-simple-core-data-driven-ios-app/<br/>
+***Native iOS RealmSwift Documentation (a more beginner-friendly alternative to Core Data)***<br/>
+https://realm.io/docs/swift/latest/<br/>
+***Native iOS RealmSwift Tutorial***<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:176]:
```suggestion
***Native iOS Realm Swift Tutorial***<br/>
```

File: mobile-apps/README.md [hunk 8]
```diff
@@ -1,4 +1,191 @@
...
+In some cases, your app might need to store local data that pertains only to the user of your app. You can use a local database for this, and there a variety of options listed below.  If you only need to store a few properties as key-value pairs, manipulating a list of user preferences is a much lighter option than creating a fully-fledged database.<br/>
+Keep in mind that if you want your user’s data to be easily accessible by other users or by other applications, using an online database is a much better solution. Check out our database section on options for getting this set up. 
+
+**Database Options**<br/>
+These are just a few of the many database options available to use.<br/>
+***Native iOS Core Data Documentation: Core Data is Apple's official local database solution***<br/>
+https://developer.apple.com/documentation/coredata<br/>
+***Native iOS Core Data Tutorial***<br/>
+http://www.brianadvent.com/build-simple-core-data-driven-ios-app/<br/>
+***Native iOS RealmSwift Documentation (a more beginner-friendly alternative to Core Data)***<br/>
```
Comment (didizhou1) [inline mobile-apps/README.md:174]:
```suggestion
***Native iOS Realm Swift Documentation (a more beginner-friendly alternative to Core Data)***<br/>
```

**Comment by Gamebot2** [comment]
Hot damn. Looks great to me.

**Pull request merged**